playbooks/roles/caasp4/templates/skuba-terraform-libvirt.tfvars.j2

```diff
@@ -25,6 +25,15 @@ repositories = {
   serverapps_updates = "http://download.suse.de/ibs/SUSE/Updates/SLE-Module-Server-Applications/15-SP1/x86_64/update/"
 }
 
+lb_repositories = {
```

Comment:
Do you plan to update the CI example from https://github.com/SUSE/skuba/blob/master/ci/infra/libvirt/terraform.tfvars.json.ci.example , too?

Reply:
It's there now: https://github.com/SUSE/skuba/pull/622